src/state/apis/swappers/swappersApi.ts

```diff
@@ -39,7 +40,7 @@ export const swappersApi = createApi({
               swapperName: SwapperName.LIFI,
             })
           } catch (error) {
-            console.error('[getLifiTradeQuoteHelper]', error)
+            console.error(error)
```

Comment:
no need for prefixing console error, now that `logger` is binned we actually get stack traces